common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -26,129 +26,135 @@ public class CommonValue {
     public static String WARM_BOX_TYPE = "warm";
 
     public static final String ML_MODEL_INDEX = ".plugins-ml-model";
+    public static final String ML_MODEL_RELOAD_INDEX = ".plugins-ml-model-reload";
```

Comment:
Where is the schema of this index?

Reply:
the method of creating index and saving data can be seen in  MLModelAutoReLoader#saveLatestReTryTimes